fastlane/lib/fastlane/command_line_handler.rb

```diff
@@ -5,7 +5,13 @@ class CommandLineHandler
     # @param [Array] args A hash of all options (e.g. --env NAME)
     def self.handle(args, options)
       lane_parameters = {} # the parameters we'll pass to the lane
-      platform_lane_info = [] # the part that's responsible for the lane/platform definition
+      platform_lane_info = args.slice!(0, 1) # the part that's responsible for the lane/platform definition
+      platform_name = platform_lane_info[0]
+      if platform_name && SupportedPlatforms.all.include?(platform_name.to_sym)
```

Comment:
Does `SupportedPlatforms.all` include the platforms that were added at runtime via https://docs.fastlane.tools/actions/add_extra_platforms/?

Reply:
Hmm, seems `SupportedPlatforms.all` does not return extra platforms in here :(